### PR TITLE
Squashed migrations into 1 and added required indexes

### PR DESCRIPTION
**Pull request by shreyanshss7**
- Django automatically creates an index on 
       - unique fields
       - primary key fields
       - foreign key fields
- Also, by default django creates a not null constraint on all columns.

### CODE REVIEW

File: fyle_slack_app/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.1.6 on 2021-02-17 09:53
+# Generated by Django 3.1.6 on 2021-03-23 10:16
```
Comment (shreyanshss7) [inline fyle_slack_app/migrations/0001_initial.py:1]: - Django automatically creates an index on 
       - unique fields
       - primary key fields
       - foreign key fields
- Also, by default django creates a not null constraint on all columns.

Comment (shreyanshss7) [reply fyle_slack_app/migrations/0001_initial.py:1]: Attached table def screenshots from db in the [task](https://app.clickup.com/t/2fbcd2)

File: fyle_slack_app/migrations/0001_initial.py [hunk 2]
```diff
@@ -29,10 +29,12 @@ class Migration(migrations.Migration):
...
             name='User',
             fields=[
-                ('slack_user_id', models.CharField(max_length=120, primary_key=True, serialize=False)),
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('slack_user_id', models.CharField(max_length=120, unique=True)),
                 ('slack_dm_channel_id', models.CharField(max_length=120, unique=True)),
                 ('email', models.EmailField(max_length=254)),
-                ('fyle_refresh_token', models.TextField()),
+                ('fyle_refresh_token', models.TextField(db_index=True)),
+                ('fyle_employee_id', models.CharField(max_length=120)),
```
Comment (abhishek1234321) [inline fyle_slack_app/migrations/0001_initial.py:37]: is this `org_user_id`? isn't this `unique`?

**Review by abhishek1234321 — APPROVED** [review]

**Review by abhishek1234321 — APPROVED** [review]

File: fyle_slack_app/migrations/0001.sql [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+create table report_polling_details (
+    id serial primary key,
+    user_id character varying(120) not null references users(slack_user_id) on delete cascade,
+    last_successful_poll_at timestamp with time zone NOT NULL,
+    created_at timestamp with time zone NOT NULL,
+    updated_at timestamp with time zone NOT NULL
+);
+
+create index idx_report_polling_details_user_id on report_polling_details using btree (user_id);
+create index idx_report_polling_details_last_successful_poll_at on report_polling_details using btree (last_successful_poll_at);
```
Comment (shreyanshss7) [inline fyle_slack_app/migrations/0001.sql:37]: Not creating indexes on unique columns since indexes are automatically created on unique columns.

**Review by abhishek1234321 — APPROVED** [review]

**Pull request closed without merging**